### PR TITLE
re-introduce comments into the AST as a node

### PR DESCRIPTION
**Pull request by thysultan**
the changes in src/Tokenizer.js are mostly moving functions around(re-arranging so to speak)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,14 @@ const declaration = {
 	line: 1, column: 1
 }
 
+const comment = {
+	value: '/*@noflip*/',
+	type: 'comm',
+	props: '/',
```
Comment (Andarist) [inline README.md]: what are possible `props` by design? by running this:
```js
compile(`.user {
  --foo: 10px; // test
}`)
```
I see that inline comment is normalized to a block comment (getting `value: '/* test*/'`) which is fine by me as discussed previously. I'm only wondering if `props: '\n'` is by design for this case.

File: src/Tokenizer.js [hunk 2]
```diff
@@ -19,6 +19,13 @@ export function node (value, root, type, props, children, length) {
 	return {value: value, root: root, type: type, props: props, children: children, line: line, column: column, length: length, prefix: '', return: ''}
 }
 
+/**
+ * @return {number}
+ */
+export function char () {
```
Comment (Andarist) [inline src/Tokenizer.js]: maybe `current` would be better for this?

File: README.md (hunk 1 above)
Comment (thysultan) [reply README.md]: Yes, `/` is for block comments and '\n' for inline comments noted by the last character  used to terminate them.

Comment (Andarist) [reply README.md]: gotcha, cool 👍

**Pull request merged**